ibis/expr/operations/core.py

```diff
@@ -31,6 +31,10 @@ def op(self):
     def to_expr(self):
         ...
 
+    def __repr__(self) -> str:
+        cls = self.__class__
+        return f"<{cls.__module__}.{cls.__name__} at 0x{hex(id(self))}>"
```

Comment:
The following may be simpler. Also added a comment.

```suggestion
    # Avoid custom repr for performance reasons
    __repr__ = object.__repr__
```